common/djangoapps/student/forms.py

```diff
@@ -396,16 +394,29 @@ def clean_email(self):
             )
         return email
 
-    def clean_year_of_birth(self):
-        """
-        Parse year_of_birth to an integer, but just use None instead of raising
-        an error if it is malformed
-        """
-        try:
-            year_str = self.cleaned_data["year_of_birth"]
-            return int(year_str) if year_str is not None else None
-        except ValueError:
-            return None
+    def clean_org_name(self):
+        """ Enforce organization related field conditions"""
+        clean_org_name = self.cleaned_data.get("org_name")
+        clean_org_size = self.cleaned_data.get("org_size")
+        clean_org_type = self.cleaned_data.get("org_type")
```

Comment:
How about putting this code in clean method

> The form subclass’s clean() method can perform validation that requires access to multiple form fields. This is where you might put in checks such as “if field A is supplied, field B must contain a valid email address”. This method can return a completely different dictionary if it wishes, which will be used as the cleaned_data.


https://docs.djangoproject.com/en/1.11/ref/forms/validation/#form-and-field-validation

Reply:
`clean_<name>` is also a django convention which helps you avoid a bloated `clean()` method, being used for all the field validations.

Reply:
It is recommended way in our case, because we accessing multiple fields